.github/workflows/scripts/ti_build/entry.py

```diff
@@ -51,11 +51,16 @@ def build_wheel(python: Command, pip: Command) -> None:
     elif wheel_tag:
         proj_tags.extend(["egg_info", f"--tag-build={wheel_tag}"])
 
-    if platform.system() == "Linux":
+    u = platform.uname()
+    if (u.system, u.machine) == ("Linux", "x86_64"):
         if is_manylinux2014():
             extra.extend(["-p", "manylinux2014_x86_64"])
         else:
-            extra.extend(["-p", "manylinux_2_27_x86_64"])
+            extra.extend(["-p", "manylinux_2_28_x86_64"])
```

Comment:
Why the ABI change?
libc 2.27 is shipped with Ubuntu 18.04 and (for compatibility) we shouldn't upgrading it unless with solid reasons.

Reply:
because 2014 and 2_27 will disappear during 2025 to 2_28